etc/mtree/BSD.lib64cb.dist

```diff
@@ -0,0 +1,27 @@
+#
+# Please see the file src/etc/mtree/README before making changes to this file.
+#
+
+/set type=dir uname=root gname=wheel mode=0755
+.
+    lib64cb
+        c18n
+        ..
```

Comment:
Given that c18n isn't supported with the benchmark ABI, I wonder if it should be added later instead.

Reply:
Given MACHINE_ABI:Mpurecap gates c18n for the native ABI this is consistent with that

Reply:
(and we do need c18n support)